dotcom-rendering/src/components/SelfHostedVideo.importable.tsx

```diff
@@ -656,17 +684,25 @@ export const SelfHostedVideo = ({
 		}
 	};
 
+	const aspectRatio = width / height;
+
 	const AudioIcon = isMuted ? SvgAudioMute : SvgAudio;
 
 	const optimisedPosterImage = showPosterImage
 		? getOptimisedPosterImage(posterImage)
 		: undefined;
 
 	return (
-		<div css={videoAndBackgroundStyles(isCinemagraph)}>
+		<div
+			css={videoContainerStyles(
+				isCinemagraph,
+				aspectRatio,
+				containerAspectRatio,
```

Comment:
nit: could the two ratio params be aligned with the naming convention of the videoContainerStyles args? Perhaps updating videoContainerStyles args to be videoAspectRatio -> aspectRatio and aspectRatio -> containerAspectRatio ?